src/manifold/src/quickhull.h

```diff
@@ -780,51 +321,15 @@ class QuickHull {
   //   distance to a plane to consider a point being on positive side of it (for
   //   a point cloud with scale 1)
   // Returns:
-  //   Convex hull of the point cloud as a mesh object with half edge structure.
-  HalfEdgeMesh getConvexHullAsMesh(const double* vertexData, size_t vertexCount,
-                                   bool CCW, double eps = defaultEps());
+  //   Convex hull of the point cloud as halfEdge vector and vertex vector
+  ConvexHull getConvexHullAsMesh(const Vec<glm::dvec3>& pointCloud, bool CCW,
```

Comment:
Questions: Is it useful to keep CCW here? Can we pass the mesh epsilon from our mesh function?

No need to fix it in this PR, just food for thought.